rotkehlchen/data_migrations/migrations/migration_9.py

```diff
@@ -24,7 +24,12 @@ def _reset_curve_decoded_events(write_cursor: 'DBCursor') -> None:
     The code is taken from `delete_events_by_tx_hash` right before 1.27 release and is
     modified to delete only transactions that have a decoded curve event.
     """
-    write_cursor.execute('SELECT DISTINCT tx_hash from evm_transactions LEFT JOIN history_events ON history_events.event_identifier=tx_hash WHERE counterparty IS ?', (CPT_CURVE,))  # noqa: E501
+    query = (
+        'SELECT DISTINCT tx_hash from evm_transactions LEFT JOIN history_events '
+        'ON history_events.event_identifier=tx_hash INNER JOIN evm_events_info ON '
+        'history_events.identifier=evm_events_info.identifier WHERE counterparty IS ?'
+    )
+    write_cursor.execute(query, (CPT_CURVE,))
```

Comment:
no need to put it in its own variable if you still split it in multiple lines. Just do it directly as before but also splitting in multiple lines from inside `cursor.execute()`